contrib/ansible/roles/objectstorage/tasks/main.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Set facts
+  set_fact:
+    minio_path: "/opt/minio"
```

Comment:
place in  `/usr/bin/minio`

Reply:
changed to /usr/local/bin/minio, as desired by the minio.service